src/app/components/VideoPlayer/index.js

```diff
@@ -147,20 +188,20 @@ const VideoPlayer = ({
   videoEl.addEventListener('pause', () => {
     clearTimeout(fuzzyTimeout);
 
-    if (videoControlsEl && videoControlsEl.api.isScrubbing()) {
+    if (videoControlsEl && videoControlsEl.api?.isScrubbing()) {
       return;
     }
 
     videoEl.setAttribute('paused', '');
 
     if (videoControlsEl) {
-      videoControlsEl.api.setPlaybackLabel('Play');
+      videoControlsEl.api?.setPlaybackLabel('Play');
     }
   });
 
   videoEl.addEventListener('play', nextFuzzyIncrement);
-  videoEl.addEventListener('stalled', clearTimeout(fuzzyTimeout));
-  videoEl.addEventListener('waiting', clearTimeout(fuzzyTimeout));
+  videoEl.addEventListener('stalled', () => clearTimeout(fuzzyTimeout));
+  videoEl.addEventListener('waiting', () => clearTimeout(fuzzyTimeout));
```

Comment:
Good catch.

Reply:
It's this kind of thing that keeps me coming back to typescript. Would not have caught it without type checks.